internal/fnruntime/runner.go

```diff
@@ -74,41 +75,46 @@ func NewRunner(
 		}
 	}
 	if fltr.Run == nil {
-		switch {
-		case f.Image != "":
-			cfn := &ContainerFn{
-				Path:            pkgPath,
-				Image:           f.Image,
-				ImagePullPolicy: imagePullPolicy,
-				Ctx:             ctx,
-				FnResult:        fnResult,
+		if f.Image == "builtins/gen-pkg-context" {
```

Comment:
what happens if users configure "builtins/gen-pkg-context"  in the `pipeline` field and potentially provide the flag to include meta resources to the function? (IIUC, the Kptfile resource is converted the pkgContext and excluded from the input resources.

Reply:
Right, I'm afraid the builtin/gen-pkg-context changes the way how `--include-meta-resources` flag works unexpectedly: The `include-meta-resources` flag means some functions require the Kptfile, but  `builtins/gen-pkg-context` converts the kptfile to something else (pkg-context).  Is there a way we can prevent that from happening?